parachains/runtimes/bridge-hubs/bridge-hub-rococo/src/lib.rs

```diff
@@ -597,6 +601,14 @@ mod benches {
 		// NOTE: Make sure you point to the individual modules below.
 		[pallet_xcm_benchmarks::fungible, XcmBalances]
 		[pallet_xcm_benchmarks::generic, XcmGeneric]
+		// Bridge pallets at Rococo
+		[pallet_bridge_grandpa, BridgeWococoGrandpa]
```

Comment:
@svyatonik 
I dont know how does this benchmarks macro magic works in deep :D, 
but I am not sure, if this prefix `[pallet_bridge_grandpa` can be the same for Rococo and Wococo:

so, maybe instead of this:
```
[pallet_bridge_grandpa, BridgeWococoGrandpa]
[pallet_bridge_grandpa, BridgeRococoGrandpa]
```
try something like this:
```
[pallet_bridge_grandpa::wococo, BridgeWococoGrandpa]
[pallet_bridge_grandpa::rococo, BridgeRococoGrandpa]
```

and after that, these commands should work and generate weight files to this PR directly (just add this commands as PR comments)
```
/cmd queue -c bench $ pallet bridge-hub-rococo bridge-hubs pallet_bridge_grandpa::wococo
/cmd queue -c bench $ pallet bridge-hub-rococo bridge-hubs pallet_bridge_grandpa::rococo
/cmd queue -c bench $ pallet bridge-hub-rococo bridge-hubs pallet_bridge_parachains::wococo
/cmd queue -c bench $ pallet bridge-hub-rococo bridge-hubs pallet_bridge_parachains::rococo
/cmd queue -c bench $ pallet bridge-hub-rococo bridge-hubs pallet_bridge_messages::wococo
/cmd queue -c bench $ pallet bridge-hub-rococo bridge-hubs pallet_bridge_messages::rococo
```

and when those weights files are pushed, we should change imports:
```
// from bridges repo imports
type WeightInfo = pallet_bridge_grandpa::weights::BridgeWeight<Runtime>;
// to generated file imports
type WeightInfo = weights::pallet_bridge_grandpa::BridgeWeight<Runtime>;
```

and also, to support weight regeneration on cumulus releases, we need to add those `pallet_bridge_grandpa::wococo...` here:
https://github.com/paritytech/cumulus/blob/bridge-benchmarks-for-bridge-hub-rococo-wococo/scripts/benchmarks-ci.sh#L42

Reply:
@svyatonik 
ok, very cool, so it is working without the change `pallet_bridge_grandpa::wococo`,
so I guess, we just need to fix only `type WeightInfo =` and that `benchmarks-ci.sh` and we are done here

Reply:
@bkontur 
Do you know a way to test it locally? :) As I said [here](https://github.com/paritytech/cumulus/pull/2107#discussion_r1081126081), I'm getting XCM-related compilation errors when trying to run `reset && cargo build --profile production --features runtime-benchmarks -p bridge-hub-rococo-runtime`. IDK - is it me, or my CLI or it is broken by default? :) And apparently there's some issue with messages benchmarks - https://gitlab.parity.io/parity/mirrors/cumulus/-/jobs/2301305#L147

Reply:
Thanks! Apparently I've been trying to use pallet as binary :facepalm: I see now that we need some changed in the `bridge-runtime-common` to support those benchmarks. I'll prepare a PR there first